airflow_kubernetes_job_operator/job_runner.py

```diff
@@ -156,17 +156,14 @@ def custom_prepare_job_kind(cls, body: dict):
             "Cannot create a job without a template, 'spec.template' is missing or not a dictionary"
         )
 
-        assert isinstance(descriptor.metadata["template"].get("spec", None), dict), JobRunnerException(
+        assert isinstance(descriptor.spec.get("template", {}).get("spec", None), dict), JobRunnerException(
```

Comment:
The default value (`{}`) in `descriptor.spec.get("template", {})` is useless, no? If template is not defined, the second get will fail regardless.

Reply:
Yea. Correct. Better standard though for what I'm checking.